requirements_docs.txt

```diff
@@ -6,4 +6,4 @@ sphinxcontrib-websupport
 pytest-sphinx
 git+git://github.com/rtfd/sphinx-notfound-page@master
 sphinx-copybutton
-sphinx-gallery
+git+git://github.com/larsoner/sphinx-gallery@scrape
```

Comment:
This needs to be changed to a released version of Sphinx Gallery after https://github.com/sphinx-gallery/sphinx-gallery/pull/494 is complete